src/molecules/Footer/footer.module.css

```diff
@@ -0,0 +1,12 @@
+footer {
+  padding: 20px 28px;
+  background-color: var(--footer-background-color);
+  display: flex;
+  justify-content: space-between;
+}
+
+footer a {
```

Comment:
Please change this to `footer a:any-link, footer a:link, footer a:visited`.

Reply:
Done.